tests/dataimportcron_test.go

```diff
@@ -72,6 +74,23 @@ var _ = Describe("DataImportCron", Serial, func() {
 
 		By("[AfterEach] Restore the profile")
 		Expect(utils.UpdateStorageProfile(f.CrClient, scName, *originalProfileSpec)).Should(Succeed())
+
+		// Clean up existing dataimportcrons in the environment that we might have switched
+		l, err := labels.Parse(common.DataImportCronLabel)
+		Expect(err).ToNot(HaveOccurred())
+		snapshots := &snapshotv1.VolumeSnapshotList{}
+		err = f.CrClient.List(context.TODO(), snapshots, &client.ListOptions{Namespace: metav1.NamespaceAll, LabelSelector: l})
+		Expect(err).To(Or(
+			Not(HaveOccurred()),
+			Satisfy(meta.IsNoMatchError),
+		))
+		for i := range snapshots.Items {
+			err = f.CrClient.Delete(context.TODO(), &snapshots.Items[i])
+			Expect(err).To(Or(
+				Not(HaveOccurred()),
+				Satisfy(meta.IsNoMatchError),
+			))
+		}
```

Comment:
Question about the scope of this cleanup.  It is being applied to all tests.  Why not delete PVCs?  Is only one test or set of tests requiring snapshot cleanup?  If so it may be better to limit this cleanup to a sub context

Reply:
Most tests in this suite have to go through this clean up, it's a pretty cheap op so don't think we care too much about deleting them in between

Reply:
Why not explicitly delete the PVCs then?

Reply:
We can, there's just no reason to since we allow switching from pvc->snap sources

Reply:
Yeah `snap->pvc` transition not supported automatically